scripts/install

```diff
@@ -129,19 +132,11 @@ ynh_script_progression --message="Configuring a systemd service..." --weight=1
 # Create a dedicated systemd config
 ynh_add_systemd_config
 
-#=================================================
-# SETUP LOGROTATE
-#=================================================
-ynh_script_progression --message="Configuring log rotation..." --weight=1
-
-# Use logrotate to manage application logfile(s)
-ynh_use_logrotate
-
 #=================================================
 # ADVERTISE SERVICE IN ADMIN PANEL
 #=================================================
 
-yunohost service add $app --description="Low-code programming for event-driven applications" --log="/var/log/$app/$app.log"
+yunohost service add $app --description="Low-code programming for event-driven applications" --log_type="systemd"
```

Comment:
Thanks for taking into account #49!

A little question: you mentioned that this is taken from the `example_ynh`, I wonder what is most compliant regarding the Yunohost standard.

I wonder if:
 - my request is legit and leave it as is;
 - if I should suggest to globalise this change through an issue in `example_ynh` ;
 - my initial request is actually a bad idea;

(mayhaps i scratch a little too much y head for nothing ^^)

Reply:
Let's try it, it does not cost much to reverse it if we discover it hinders debuggability. :)